kubeflow/compiled/evaluate_the_model.yaml

```diff
@@ -136,7 +136,7 @@ spec:
     parameters:
     - {name: docker, value: 'unitytechnologies/datasetinsights:latest'}
     - {name: source_uri, value: 'https://storage.googleapis.com/datasetinsights/data/groceries/v3.zip'}
-    - {name: config, value: datasetinsights/configs/faster_rcnn_groceries_real.yaml}
+    - {name: config, value: 'file://datasetinsights/configs/faster_rcnn_groceries_real.yaml'}
```

Comment:
Can you change the default value to absolute path. We discussed dropping support for relative path.